README.md

```diff
@@ -31,16 +32,14 @@ Find out more on our [mission and scope](https://movement.neuroinformatics.dev/c
 
 ## Status
 > [!Warning]
-> - 🏗️ The package is currently in early development. Stay tuned ⌛
-> - It is not sufficiently tested to be used for scientific analysis
-> - The interface is subject to changes.
+> 🏗️ The package is currently in early development and the interface is subject to change. Feel free to play around and provide feedback.
 
 ## Join the movement
 
 Contributions to movement are absolutely encouraged, whether to fix a bug, develop a new feature, or improve the documentation.
 To help you get started, we have prepared a detailed [contributing guide](https://movement.neuroinformatics.dev/community/contributing.html).
 
-You are welcome to chat with the team on [zulip](https://neuroinformatics.zulipchat.com/#narrow/stream/406001-Movement/topic/Welcome!). You may also [open an issue](https://github.com/neuroinformatics-unit/movement/issues) to report a bug or request a new feature.
+You are welcome to chat with the team on [zulip](https://neuroinformatics.zulipchat.com/#narrow/stream/406001-Movement). You may also [open an issue](https://github.com/neuroinformatics-unit/movement/issues) to report a bug or request a new feature.
```

Comment:
I know this wasn't changed in this PR, but I noticed the "may" and for some reason it seems a bit formal.
```suggestion
You are welcome to chat with the team on [zulip](https://neuroinformatics.zulipchat.com/#narrow/stream/406001-Movement). You can also [open an issue](https://github.com/neuroinformatics-unit/movement/issues) to report a bug or request a new feature.
```